src/components/article/PostArticle.jsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import propTypes from "prop-types";
+import TagsInput from "react-tagsinput";
+import "react-tagsinput/react-tagsinput.css";
+
+import createArticles from "../../actions/articleActions";
+
+class PostArticle extends Component {
+  state = {
+    title: "",
+    imageUrl: "",
+    description: "",
+    body: "",
+    audioUrl: "",
+    tags: []
+  };
+
+  onChange = e => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  onChang = tags => {
```

Comment:
Could you rename the method to indicate what it does exactly

Reply:
@shakirandagire I have worked on this